java/gradle/dse/build.gradle

```diff
@@ -0,0 +1,33 @@
+group 'com.datastax.spark.example'
+version '0.1'
+apply plugin: 'java'
+
+repositories {
+    mavenCentral()
+    maven {
+        url "https://datastax.artifactoryonline.com/datastax/public-repos/"
+    }
+}
+
+configurations {
```

Comment:
This example `jar` command is going to technically build a fat jar incase that wasn't clear :)